core/src/Streamly/Internal/Data/Stream/StreamD/Exception.hs

```diff
@@ -272,9 +272,13 @@ bracket' :: (MonadAsync m, MonadCatch m) =>
     -> (b -> m e)
     -> (b -> Stream m a)
     -> Stream m a
-bracket' bef aft exc gc =
-    gbracket bef (inline MC.try) aft gc
+bracket' bef aft gc exc =
```

Comment:
Why did the order change in this? This was not proposed. This is dangerous because this will silently introduce bugs in all current users of `bracket'` . If we change the order here we should fix all the callers of this. Because of this, IsStream.bracket' behaviour would become incorrect, streamly-process package may also be broken by this.

Reply:
Let's not change the order in this API.